job/server/src/main/java/alluxio/master/job/plan/PlanTracker.java

```diff
@@ -178,84 +124,17 @@ public PlanCoordinator getCoordinator(long jobId) {
    * @throws ResourceExhaustedException if there is no more space available in the job master
    */
   public synchronized void run(PlanConfig jobConfig, CommandManager manager,
-      JobServerContext ctx, List<WorkerInfo> workers, long jobId) throws
+                               JobServerContext ctx, List<WorkerInfo> workers, long jobId) throws
       JobDoesNotExistException, ResourceExhaustedException {
     checkActiveSetReplicaJobs(jobConfig);
-    if (removeFinished()) {
+    if (mCoordinators.size() < mCapacity) {
```

Comment:
Why is there the capacity limit. the running job limits is here:
https://github.com/Alluxio/alluxio/blob/1359cc53018413e28044152a24d0c762b300f25b/job/server/src/main/java/alluxio/master/job/tracker/DistLoadCliRunner.java#L159

Reply:
Compatibility with some unit tests.